nt-be/src/handlers/token/price.rs

```diff
@@ -109,15 +103,13 @@ async fn fetch_ft_price(
 
     match http_client.get(&url).send().await {
         Ok(response) => {
-            if let Ok(data) = response.json::<NearBlocksTokenResponse>().await {
-                if let Some(contract) = data.contracts.first() {
-                    if let Some(price_str) = &contract.price {
-                        if let Ok(price) = price_str.parse::<f64>() {
-                            println!("✓ Token {} price from Nearblocks: ${}", token_id, price);
-                            return Ok((price, "nearblocks".to_string()));
-                        }
-                    }
-                }
+            if let Ok(data) = response.json::<NearBlocksTokenResponse>().await
```

Comment:
Please sync with Megha on it as she reverted this before (I believe she is using an older rust version)
@Megha-Dev-19 can you update your rust version?